app/src/main/java/space/taran/arknavigator/mvp/model/repo/kind/DocumentKindFactory.kt

```diff
@@ -11,7 +11,7 @@ import java.nio.file.Path
 
 object DocumentKindFactory : ResourceKindFactory<ResourceKind.Document> {
     override val acceptedExtensions: Set<String> =
-        setOf("pdf", "txt", "doc", "docx", "odt", "ods", "md")
+        setOf("pdf", "txt", "doc", "docx", "odt", "ods", "md", "zip")
```

Comment:
We need new kind of resources for archives. It would include `zip` and also other types like `rar`, `7z`, `tar.gz`.

Reply:
See https://github.com/ARK-Builders/ARK-Navigator/pull/229/files#diff-104e24380170e5848ab9a1bff4d399bbb1bc30ab9c68e5db1ed1c26051b0e14bR1

Reply:
I will look into this.